aces/Loris
# [instrument_list] Change check for submitted and populate RB

## Brief summary of changes
Addresses: https://github.com/aces/Loris/issues/5701 in 3 different ways

1. The RB data had `NULL` values under the `Submitted` column. however the default in Timepoint class is set to `N` [See Timpoint class](https://github.com/aces/Loris/blob/22.0-release/php/libraries/TimePoint.class.inc#L248). This PR adds the `N` value wherever `Submitted=NULL`
2. The check for submitted in the instrument_list class was done through a double negation instead of simply using the `isSubmitted()` function.
3. The default in SQL for `Submitted` is NULL and thus the database does not enforce the rules in the timepoint class. This PR adds the SQL part. 

#### Testing instructions (if applicable)
1. try to reproduce error in the linked issue by setting the Submitted column to null for a session and see if the data can be submitted.
